.github/workflows/build.yml

```diff
@@ -16,8 +16,7 @@ jobs:
           - macos-latest  # aarch64-darwin
           - ubuntu-latest # x86_64-linux
         ghc:
-          - ghc8107
-          - ghc902
+          - ghc966
```

Comment:
@DavidMazarro If we are only going to use one version of GHC in the future, I believe we can remove the GHC version from the package name:

```diff
- nix run .#test-ghc966
+ nix run .#test
```